img/megafon.svg

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 381.977 381.977" style="enable-background:new 0 0 381.977 381.977;" xml:space="preserve">
+<g>
+	<g>
+		<path d="M288.72,122.188c-1.804-2.779-5.52-3.57-8.299-1.766c-2.78,1.804-3.57,5.52-1.766,8.299
+			c0.364,0.56,0.819,1.056,1.346,1.467c18.441,19.69,18.441,50.31,0,70c-2.222,2.398-2.115,6.133,0.24,8.4
+			c2.343,2.34,6.137,2.34,8.48,0C311.48,184.285,311.48,146.492,288.72,122.188z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M317.6,99.868c-1.804-2.78-5.52-3.57-8.299-1.766s-3.57,5.52-1.766,8.299c0.364,0.56,0.819,1.056,1.346,1.467
+			c30.005,32.077,30.005,81.923,0,114c-1.804,2.78-1.013,6.495,1.766,8.299c2.145,1.392,4.937,1.272,6.954-0.299v0.48
+			C351.973,193.646,351.973,136.57,317.6,99.868z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M346.64,75.388c-2.276-2.408-6.074-2.515-8.482-0.238c-0.082,0.077-0.161,0.156-0.238,0.238
+			c-2.34,2.343-2.34,6.137,0,8.48c42.787,45.71,42.787,116.77,0,162.48c-2.614,2.036-3.083,5.806-1.046,8.421
+			c2.036,2.614,5.806,3.083,8.421,1.046c0.527-0.411,0.982-0.906,1.346-1.467C393.755,203.998,393.755,125.738,346.64,75.388z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M226.96,38.508c-3.314,0-6,2.686-6,6v14.32L12,113.628v-17.12c0-3.314-2.686-6-6-6c-3.314,0-6,2.686-6,6v130.24
+			c0,3.314,2.686,6,6,6c3.314,0,6-2.686,6-6v-17.12l37.04,9.76l40.96,120c0.823,2.437,3.108,4.078,5.68,4.08h57.28
+			c3.314,0.013,6.01-2.663,6.023-5.977c0.003-0.689-0.113-1.374-0.343-2.023l-33.04-96l95.36,24.96v14.32c0,3.314,2.686,6,6,6
+			s6-2.686,6-6V44.508C232.96,41.194,230.274,38.508,226.96,38.508z M144.56,331.548H99.92l-36.8-108.08l48.72,12.8L144.56,331.548z
+			 M220.96,252.028L12,197.148v-71.2l208.96-54.72V252.028z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M181.991,102.145c-0.611-3.257-3.747-5.402-7.004-4.791c-0.336,0.063-0.666,0.155-0.987,0.274l-124.72,34.8
+			c-3.181,0.927-5.009,4.257-4.082,7.439c0.738,2.533,3.044,4.287,5.682,4.321l0.08,0.08h1.6l124.64-35.12
+			C180.457,108.537,182.602,105.402,181.991,102.145z"/>
+	</g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>

```

Comment:
You can use a tool like https://jakearchibald.github.io/svgomg/ to clean this .svg files